ios/.gitignore

```diff
@@ -0,0 +1,30 @@
+# OSX
```

Comment:
how that folder went into this PR ?

Reply:
because I cleaned my prebuild locally, due to changes I made to packages.
the prebuild allows me to build from my PC to phone directly

Reply:
But there still others .cpp files added to this PR that make sens